my-app/src/api/mountainsAPI.js

```diff
@@ -0,0 +1,12 @@
+export const fetchMountainList = async () => {
+  try {
+    const resp = await fetch(
+      "https://my-json-server.typicode.com/mihul87/mockjson/mountains"
```

Comment:
fain. poti la fel de bine sa instalezi si pachetul de json-server ce il vei rula in pe alt port, iar in proiectul tau vei adauga un proxy in package.json
Astfel documentul de json (datele) le vei avea local.

Reply:
Bine!